content/intermediate/templates/conditions/index.md

```diff
@@ -109,31 +109,76 @@ Let’s deploy the solution!
 
 When you create the stack, you will pass `test` as the value for `EnvType`, and you will observe only an EC2 instance resource will be provisioned by CloudFormation. Save the template you have updated with content above; next, navigate to the AWS CloudFormation [console](https://console.aws.amazon.com/cloudformation), and choose to create a stack using this template:
 
-* In the CloudFormation console, select **Create stack**, **With new resources (standard)**.
-* In **Prepare template**, select **Template is ready**.
-* In **Template source**, select **Upload a template file**.
-* Choose the `condition-resource.yaml` template.
-* Enter a **Stack name**. For example, choose to specify `cfn-workshop-condition-test`.
-* Pass `test` as the value for the `EnvType` parameter. Choose **Next**.
-* Choose to accept default values in the **Configure stack options** page. Choose **Next**.
-* Choose **Create stack**. You can view the progress of the stack being created in the CloudFormation console.
-* Wait until the stack creation is complete. Refresh the view in the console until you see your stack to be in the `CREATE_COMPLETE` status.
-
+:::::tabs{variant="container"}
+::::tab{id="cloud9" label="Cloud9"}
+1. Upload the file to your **template S3 bucket** using AWS CLI [aws s3 cp](https://docs.aws.amazon.com/cli/latest/reference/s3/cp.html) command.
+
+   `aws s3 cp code/workspace/condition-resource.yaml s3://cfn-workshop-01-{accountid}`
```

Comment:
Can we please use code blocks to make it clearer and neater?
`:::code{language=shell showLineNumbers=false showCopyAction=true}
aws s3 cp code/workspace/condition-resource.yaml s3://cfn-workshop-01-{accountid}
:::`